pkg/templates/pdbmaxunavailable/template.go

```diff
@@ -0,0 +1,53 @@
+package pdbmaxunavailable
+
+import (
+	"golang.stackrox.io/kube-linter/pkg/check"
+	"golang.stackrox.io/kube-linter/pkg/config"
+	"golang.stackrox.io/kube-linter/pkg/diagnostic"
+	"golang.stackrox.io/kube-linter/pkg/lintcontext"
+	"golang.stackrox.io/kube-linter/pkg/objectkinds"
+	"golang.stackrox.io/kube-linter/pkg/templates"
+	"golang.stackrox.io/kube-linter/pkg/templates/pdbmaxunavailable/internal/params"
+	pdbV1 "k8s.io/api/policy/v1"
+)
+
+const (
+	templateKey           = "pdb-max-unavailable"
+	maxUnavailableZeroMsg = "MaxUnavailable is set to 0"
+)
+
+func init() {
+	templates.Register(check.Template{
+		HumanName:   "Pod Disruption Budget Configuration Checks",
```

Comment:
Both checks have the same strings, resulting in two sections with identical title in the generated `template.md`. How about tailoring the strings to the individual checks?

(I paused here for quite a while trying to figure out more specific strings. Having two separate checks for the two configuration settings makes it more difficult that I thought.)

How about this?

```suggestion
		HumanName:   "No pod disruption allowed - maxUnavailable",
```